satellite/console/service.go

```diff
@@ -168,6 +168,142 @@ func (s *Service) CreateUser(ctx context.Context, user CreateUser, tokenSecret R
 	return u, nil
 }
 
+// AddNewPaymentMethod adds new payment method for project
+func (s *Service) AddNewPaymentMethod(ctx context.Context, paymentMethodToken string, isDefault bool, projectID uuid.UUID) (payment *ProjectPayment, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	authorization, err := GetAuth(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	info, err := s.store.UserPayments().Get(ctx, authorization.User.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	params := payments.AddPaymentMethodParams{
+		Token:      paymentMethodToken,
+		CustomerID: string(info.CustomerID),
+	}
+
+	method, err := s.pm.AddPaymentMethod(ctx, params)
+	if err != nil {
+		return nil, err
+	}
+
+	projectPaymentInfo := ProjectPayment{
+		ProjectID:       projectID,
+		PayerID:         authorization.User.ID,
+		PaymentMethodID: method.ID,
+		CreatedAt:       time.Now(),
+		IsDefault:       isDefault,
+	}
+
+	create, err := s.store.ProjectPayments().Create(ctx, projectPaymentInfo)
+	if err != nil {
+		return nil, err
+	}
+
+	return create, nil
+}
+
+// SetDefaultPaymentMethod set default payment method for given project
+func (s *Service) SetDefaultPaymentMethod(ctx context.Context, projectPaymentID uuid.UUID, projectID uuid.UUID) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	_, err = GetAuth(ctx)
+	if err != nil {
+		return err
+	}
+
+	projectPayment, err := s.store.ProjectPayments().GetDefaultByProjectID(ctx, projectID)
```

Comment:
I think it should be one query like `db.UpdateDefault`, the rest is implementation detail.